src/extension.ts

```diff
@@ -1045,31 +1072,32 @@ async function isBashOnPod(podName : string): Promise<boolean> {
     return !result.code;
 }
 
-function syncKubernetes() {
-    selectPodForApp((pod) => {
-        if (!pod) {
-            return;
-        }
-        selectContainerForPod(pod, (container) => {
-            if (!container) {
-                return;
-            }
-            let pieces = container.image.split(':');
-            if (pieces.length !== 2) {
-                vscode.window.showErrorMessage(`unexpected image name: ${container.image}`);
-                return;
-            }
+async function syncKubernetes() : Promise<void> {
```

Comment:
i have no idea what this function is doing, and that's ok because i just realized we have https://github.com/Azure/vscode-kubernetes-tools/issues/68 to clear this up 🙃

Reply:
And there's a PR on the way for that, but it depends on this one (because of the refactoring)...